app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -225,6 +229,9 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
                 return
             }
         }
+        navigationToolbar.setChildrenClickable(false, 0)
```

Comment:
Can we throw a comment describing the problem and linking to the issue this fixes? :)